### DISCUSSION TITLE
Is there any way to change the inner access path of embedded resource files at build time?

### DISCUSSION THREAD

**Discussion by Flithor** [dotnet/roslyn#52718] [Q&A] [answered]
I want the embedded resource file not to build its inner access path based on its folder in the project, but can customize its path in some way.
the path in project:
`/Resources/MyResource.resx`
the inner access path in normal:
`Resources.MyResource.resx`
What I want:
`MyResource.resx`
`MyCustom.MyResource.resx`

**Comment by jmarolf** [comment, marked as answer]
I may not be understanding what you are trying to do but if you set the [`LogicalName`](https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-options/resources#resources) on the resource item in your project file it will be embedded with that name instead of the auto-generated name

**Reply by Flithor** [reply]
It works. But just looks like I'm on the wrong way.
